Makefile.pre.in

```diff
@@ -38,6 +38,7 @@ CC=		@CC@
 CXX=		@CXX@
 MAINCC=		@MAINCC@
 LINKCC=		@LINKCC@
+LINKCC_BUILDPYTHON =	@LINKCC_BUILDPYTHON@
```

Comment:
This is the least intrusive way I could think of to customize the link step only for building the `python.wasm` executable...

If you know of a better way, please do tell :)

Reply:
Meh! The ugly hacks are piling up. Maybe it's easier to not use ``ac_cv_pthread`` and to add ``-pthread`` to ``CFLAGS`` or ``CFLAGS_NODIST`` after all?

Reply:
Unfortunately then it would be passed to the `$(BUILDPYTHON)` target, and standalone wasm doesn't support `-pthread`